db/db_impl/db_secondary_test.cc

```diff
@@ -373,7 +373,7 @@ TEST_F(DBSecondaryTest, MissingTableFile) {
   SyncPoint::GetInstance()->DisableProcessing();
   SyncPoint::GetInstance()->ClearAllCallBacks();
   SyncPoint::GetInstance()->SetCallBack(
-      "ReactiveVersionSet::ReadAndApply:AfterLoadTableHandlers",
+      "ReactiveVersionSet::ApplyOneVersionEditToBuilder:AfterLoadTableHandlers",
```

Comment:
The original `ReactiveVersionSet::ReadAndApply:AfterLoadTableHandlers` is used in other places, e.g., `db_secondary_test.cc`.
Similar thing applies to all sync points affected in this PR. If you enable a sync point, you need to find out ALL occurrences and rename all.

Reply:
Sure. I used 'grep -r "sync point name" .' to check their occurrences.  I double checked again and I have renamed all of them.

Reply:
Sounds good!